applications/nrf_desktop/doc/config_channel.rst

```diff
@@ -127,17 +148,19 @@ To register an application module as a configuration channel listener, complete
 
 1. Make sure that the application module is an :ref:`event_manager` listener.
 #. Include the :file:`config_event.h` header.
-#. Subscribe for the ``config_event`` and ``config_fetch_request_event`` using the :c:macro:`EVENT_SUBSCRIBE` macro:
+#. Subscribe for the ``config_event`` using the :c:macro:`EVENT_SUBSCRIBE_EARLY` macro:
 
    .. code-block:: c
 
        EVENT_LISTENER(MODULE, event_handler);
        #if CONFIG_DESKTOP_CONFIG_CHANNEL_ENABLE
-       EVENT_SUBSCRIBE(MODULE, config_event);
-       EVENT_SUBSCRIBE(MODULE, config_fetch_request_event);
+       EVENT_SUBSCRIBE_EARLY(MODULE, config_event);
        #endif
 
    The module should subscribe only if the configuration channel is enabled.
+   The module must be an early subscriber to make sure it will receive the event before the configuration channel transports (:ref:`nrf_desktop_usb_state` and :ref:`nrf_desktop_hids`).
+   Otherwise, the module may not receive the configuration channel requests at all.
+   In that case an error responses will be generated by configuration channel transport.
```

Comment:
Maybe this could be a `.. note::` or a `.. tip::`?

Reply:
I will add this as `.. note ::`